experiment/autobumper/bumper/bumper.go

```diff
@@ -587,7 +595,13 @@ func makeCommitSummary(prefixes []Prefix, versions map[string][]string) string {
 	var consistentBumps []string
 	for _, prefix := range prefixes {
 		if !prefix.ConsistentImages {
-			inconsistentBumps = append(inconsistentBumps, prefix.Name)
+			for _, imageList := range versions {
+				//Should not be possible for tag to be in map with empty imageList
+				if strings.HasPrefix(imageList[0], prefix.Prefix) {
```

Comment:
This won't work if multiple prefixes bump to the same version, right? I think in that case the first entry in `imageList` may not contain the correct prefix, but a later entry would. Please add a test to cover this case as well.

Reply:
Good catch. I did not consider this possibility. Fixed and added unit test